src/helper/UserSession.cpp

```diff
@@ -195,6 +195,9 @@ namespace SDDM {
         // the session as stdin so that it stays open without races
         if (hasDisplayServer || waylandUserSession) {
             // open VT and get the fd
+            if (processEnvironment().value(QStringLiteral("XDG_VTNR")).isEmpty())
```

Comment:
If unset, it should continue, just without the VT related stuff (`vtFd = -1`)

Reply:
See https://github.com/sddm/sddm/pull/1863#discussion_r1500446883

Reply:
Ah, OK, I think I can try that

Reply:
done locally, and now it all seems to work, let me try to get these commits in order, and I will push